app/dashboard/models.py

```diff
@@ -121,41 +121,41 @@ def needs_review(self):
         """Filter results by bounties that need reviewed."""
```

Comment:
In this file, can you just include the logic changes and ditch all the style changes? I'm guessing you have some kind of automated tool in your IDE or something like that, and while I appreciate the style fix up, different contributors have different style fixing preferences in their IDEs so we get a lot of PRs where style changes are just going back and forth which makes it hard to review.

Reply:
Noted, will revert those changes, and just add the logic.